common/Services/INavigationService.cs

```diff
@@ -49,4 +49,7 @@ public static class KnownPageKeys
     public static readonly string Feedback = "DevHome.Settings.ViewModels.FeedbackViewModel";
     public static readonly string Environments = "DevHome.Environments.ViewModels.LandingPageViewModel";
     public static readonly string SetupFlow = "DevHome.SetupFlow.ViewModels.SetupFlowViewModel";
+
+    // Will not work with navigation service natively.  Used for the dictionary in SetupFlowViewModel
+    public static readonly string RepositoryConfiguration = "RepositoryConfiguration";

```

Comment:
> Would the full path not work here?
No.  When called with the full path, `GetPageType` throws because the page is not found in `_pages`  because `RepositoryConfiguration` isn't configured.

Considering the string is used in one location.  I can hard code the string.